app/views/fact_values/index.html.erb

```diff
@@ -27,4 +27,7 @@ end
   </tbody>
 </table>
 
+<span id="fact-chart-modal"></span>
+<%= mount_react_component('FactChartModal', "#fact-chart-modal") %>
```

Comment:
q: is this really required? e.g. if on Click you would open the modal, does it need a second component mounted already or can you simply open the modal and render your component inside?

Reply:
bump?